examples/gdp/batchProcessing.py

```diff
@@ -0,0 +1,230 @@
+from pyomo.environ import *
+from pyomo.gdp import *
+
+'''Problem from http://www.minlp.org/library/problem/index.php?i=172&lib=GDP
+We are minimizing the cost of a design of a plant with parallel processing units and storage tanks
+in between. We decide the number and volume of units, and the volume and location of the storage
+tanks. The problem is convexified and has a nonlinear objective and global constraints
+
+NOTE: When I refer to 'gams' in the comments, that is Batch101006_BM.gms for now. It's confusing
+because the _opt file is different (It has hard-coded bigM parameters so that each constraint 
+has the "optimal" bigM).'''
+
+model = AbstractModel()
+
+# TODO: it looks like they set a bigM for each j. Which I need to look up how to do...
```

Comment:
`j`'s are stages?

Reply:
Yes, j's are stages. I think the gams file uses 3 different Big-M values, one for each disjunction. They are indexed by stage, but I don't think they actually depend on it. I never got as far as making my Big-M values match those ones.